simulator/src/main/cpp/SimulatorFMRxTx.cpp

```diff
@@ -48,11 +49,6 @@ char* SimulatorFMRxTx::receive()
     }
 }
 
-int SimulatorFMRxTx::getToAddress(char packet[])
```

Comment:
IF this is no longer required, it should be removed from the interface, and the RFM69 class

Reply:
Removing

Reply:
it is removed from the interface. when we go to get it working on the devices we can make that change. This PR doesnt touch any files in the NRF32-RFM69 hardware folder.

Reply:
Perhaps we can raise a github issue so we do not forget